addons/addon-base-rest-api/packages/services/package.json

```diff
@@ -50,4 +50,4 @@
       "pre-commit": "pretty-quick --staged --pattern '**/*.*(js|jsx)'"
     }
   }
-}
+}
```

Comment:
Nitpick: Add extra new lines in the files so GH doesn't complain

Reply:
whenever I save the file with a new line it removes it, I will see what tool is doing that